README.md

```diff
@@ -137,4 +137,14 @@ You have to make sure you match your language code in your YAML file and the nam
 $> npm run test:selenium
 ```
 
+## Environment Variables
```

Comment:
We should add more to this as part of the general docs I just wanted to get it started here